addons/account/models/account_move_line.py

```diff
@@ -256,7 +260,7 @@ class AccountMoveLine(models.Model):
     )
     matching_number = fields.Char(
         string="Matching #",
-        readonly=True,
+        copy=False,
         help="Matching number for this line, 'P' if it is only partially reconcile, or the name of "
```

Comment:
docstring to be completed to say we can import with 'I' (or 'T' ?), although this doesn't stay

Reply:
Not sure it should be part of the tooltip, it is quite technical and an implementation detail.
The imports are managing the `I` part automatically, and it is documented in the CSV/XLS import templates

Reply:
yeah, alright... just a technical comment?
```suggestion
        help="Matching number for this line, 'P' if it is only partially reconcile, or the name of ..."  # 'Ixxxx' can also be provided during the import blablabla
 ```

Reply:
Done